### ISSUE TITLE
Codegen error for juliaset example on Mac

### ISSUE THREAD

**Issue by lkuper** [IntelLabs/ParallelAccelerator.jl#22]
Here's the log from [the Travis build](https://travis-ci.org/IntelLabs/ParallelAccelerator.jl/jobs/94523191#L389):

```
$ julia examples/juliaset/juliaset.jl
/var/folders/gw/_2jq29095y7b__wtby9dg_5h0000gn/T/tmpeJeith/cgen_output0.cpp:109:53: error: use of undeclared identifier 'cabs'
parallel_ir_temp_parallel_ir_new_array_name_3_1_1 = cabs(parallel_ir_temp_Z_1);;
                                                    ^
/var/folders/gw/_2jq29095y7b__wtby9dg_5h0000gn/T/tmpeJeith/cgen_output0.cpp:212:53: error: use of undeclared identifier 'cabs'
parallel_ir_temp_parallel_ir_new_array_name_3_1_1 = cabs(parallel_ir_temp_Z_1);;
                                                    ^
2 errors generated.
ERROR: LoadError: failed process: Process(`g++ -O3 -std=c++11 -g -fpic -c -o /var/folders/gw/_2jq29095y7b__wtby9dg_5h0000gn/T/tmpeJeith/cgen_output0.o /var/folders/gw/_2jq29095y7b__wtby9dg_5h0000gn/T/tmpeJeith/cgen_output0.cpp`, ProcessExited(1)) [1]
 in run at /Users/travis/julia/lib/julia/sys.dylib
 in compile at /Users/travis/.julia/v0.4/ParallelAccelerator/src/cgen.jl:2555
 in toCGen at /Users/travis/.julia/v0.4/ParallelAccelerator/src/driver.jl:173
 in processFuncCall at /Users/travis/.julia/v0.4/CompilerTools/src/OptFramework.jl:338
 in iterate at /Users/travis/.julia/v0.4/CompilerTools/src/OptFramework.jl:400
 in main at /Users/travis/build/IntelLabs/ParallelAccelerator.jl/examples/juliaset/juliaset.jl:121
 in include at /Users/travis/julia/lib/julia/sys.dylib
 in include_from_node1 at /Users/travis/julia/lib/julia/sys.dylib
 in process_options at /Users/travis/julia/lib/julia/sys.dylib
 in _start at /Users/travis/julia/lib/julia/sys.dylib
while loading /Users/travis/build/IntelLabs/ParallelAccelerator.jl/examples/juliaset/juliaset.jl, in expression starting on line 131
```

The Linux build seems fine.  This is our only Mac-only issue that I'm aware of at the moment.

**Comment by ninegua** [comment]
Not sure if it is clang specific, but cabs is a C99 feature that should be supported. http://en.cppreference.com/w/c/numeric/complex/cabs

**Comment by lkuper** [comment]
Maybe it's just a matter of passing the correct flag to g++, then?

This might actually be a good first bug for a volunteer contributor using a Mac with GCC.

**Comment by ninegua** [comment]
Fixed in d61ed6b42b6a9dd03e20648c07a0392690126a1a

**Comment by lkuper** [comment]
Thanks!
